eth/v1alpha1/node.proto

```diff
@@ -62,47 +47,54 @@ service Node {
             get: "/eth/v1alpha1/node/services"
         };
     }
-
-    // Retrieve the list of peers currently connected to this node.
-    rpc ListPeers(google.protobuf.Empty) returns (Peers) {
-        option (google.api.http) = {
-            get: "/eth/v1alpha1/node/peers"
-        };
-    }
-}
-
-// Information about the current network sync status of the node.
-message SyncStatus {
-    // Whether or not the node is currently syncing.
-    bool syncing = 1;
 }
 
-// Information about the genesis of Ethereum 2.0.
-message Genesis {
-    // UTC time specified in the chain start event in the deposit contract.
-    google.protobuf.Timestamp genesis_time = 1;
-
-    // Address of the deposit contract in the Ethereum 1 chain.
-    bytes deposit_contract_address = 2;
+message NodeInfo {
+    // A unique identifier for the node.
```

Comment:
peer_id would be useful as well. lighthouse seems to expose (just) peer id as an example

Reply:
Our node ID is indeed the peer ID.  We could expose the public key directly, but not sure that it would give us much over the peer ID.

Reply:
peer id is great then. making it clear that's what it is and clarifying when it can change would make it wonderful :D 
thanks